### ISSUE TITLE
A question about the paper

### ISSUE THREAD

**Issue by endNone** [OpenLMLab/GAOKAO-Bench#36]
In the paper, it mentions setting up 0-shot to mitigate the issue of data leakage. I don't quite understand the logic behind this. Could you explain it?

**Comment by zzzyyi** [comment]
Thank you for your question, Elon:).
Zero-shot means model is evaluated without seeing any task-specific examples. This can reduce the risk of overfitting and evaluate the model's generalization ability. On the other hand, for models that may have encountered GAOKAO questions during training, using few-shot settings can make them memorize this type of data, resulting in unfair results. Therefore, we believe using zero-shot settings can mitigate benchmark leakage.
Hope it can help you~

**Comment by endNone** [comment]
Hahaha:), you mean that few-shot might "make it recall similar data received during training, such as questions from the same exam paper" (my understanding)? As you mentioned, if the model received similar GAOKAO content during training, then I believe the model's accuracy should have little to do with the shot setting (I have done experiments to verify this, even a 7b model in 0-shot can achieve 90+ accuracy). Currently, the results of models in few-shot and zero-shot are generally not much different (many models now tend to perform worse in few-shot). My understanding is that the shot setting is to better help the model understand the task of multiple-choice questions, guiding it on how to answer the question? (I see it seems to have first appeared in MMLU and achieved good results with GPT-3, but now it seems to be counterproductive). If there are concerns about data leakage, can we choose some multiple-choice questions unrelated to GAOKAO content as shots? These are just my immature thoughts, and I would appreciate your guidance.

**Comment by zzzyyi** [comment]
For base model, few-shot prompt can help it understand the task, as you mentioned. On the other hand, chat model can understand the task without additional examples, so zero-shot is available. So if you want to evaluate base model and avoid benchmark leakage, choose unrelated examples as shots is considerable, from my perspective. Hope it can help:)

**Comment by endNone** [comment]
Thank you so much for your thoughtful explanation! I appreciate your insights into the use of zero-shot settings to mitigate benchmark leakage and enhance the evaluation of a model's generalization ability. Your suggestion to use unrelated examples as shots to avoid data leakage when evaluating base models is also very intriguing and seems like a practical approach. I will consider this strategy moving forward in my experiments.

I look forward to further discussions and learning more from your expertise!
